tests/kafkatest/tests/mirror_maker_test.py

```diff
@@ -13,17 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-from ducktape.tests.test import Test
+from ducktape.utils.util import wait_until
 
 from kafkatest.services.zookeeper import ZookeeperService
 from kafkatest.services.kafka import KafkaService
 from kafkatest.services.console_consumer import ConsoleConsumer
 from kafkatest.services.verifiable_producer import VerifiableProducer
 from kafkatest.services.mirror_maker import MirrorMaker
+from kafkatest.tests.produce_consume_validate import ProduceConsumeValidateTest
+
+
+def clean_bounce(test):
+    for i in range(3):
+        test.logger.info("Bringing mirror maker nodes down...")
```

Comment:
Is the intent of the test to bring down the entire service, then restart it? Or do rolling bounces? Is there any value in testing one vs the other vs both?